modules/tinymce/src/core/main/ts/api/FakeClipboard.ts

```diff
@@ -0,0 +1,89 @@
+
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+// The FakeClipboard has been designed to match the native Clipboard API as closely as possible
+// https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API
+
+import { Obj, Singleton } from '@ephox/katamari';
+
+export interface FakeClipboardItem {
+  readonly items: Record<string, any>;
+  readonly types: ReadonlyArray<string>;
+  readonly getType: <D = any>(type: string) => D | undefined;
+}
+
+/**
+  * TinyMCE FakeClipboard API.
+  *
+  * @class tinymce.FakeClipboard
+  */
+
+interface FakeClipboard {
+
+  /**
+   * Create a FakeClipboardItem instance that is used when reading or writing data via the FakeClipboard API.
+   *
+   * @method FakeClipboardItem
+   * @param {Record<string, any>} items an object with the type as the key and any data as the value
+   * @returns {FakeClipboardItem} FakeClipboardItem
```

Comment:
```suggestion
   * @returns {tinymce.FakeClipboardItem} A new fake clipboard item to represent the specified items.
```
This has to be the fully qualified name of the class.